web/src/test/java/org/bkvm/utils/AbstractBookkeeperTestUtils.java

```diff
@@ -52,22 +52,19 @@
     @Rule
     public final TemporaryFolder folder = new TemporaryFolder();
 
-    LocalZookeeperServer zkServerMain;
```

Comment:
If we use Curator we should delete this file

Reply:
good catch